Standards/Configuration/.editorconfig

```
@@ -204,13 +220,33 @@ dotnet_naming_rule.interface_types_must_be_prefixed_with_i.severity = error
 dotnet_naming_rule.interface_types_must_be_prefixed_with_i.symbols = interface_types
 dotnet_naming_rule.interface_types_must_be_prefixed_with_i.style = prefix_interface_interface_with_i
 
+csharp_style_namespace_declarations= file_scoped:warning
+csharp_using_directive_placement= outside_namespace:warning
```

```suggestion
csharp_using_directive_placement = outside_namespace:warning
```


and a question: should we make it an error too?